_components/sidenav.md

```diff
@@ -7,8 +7,6 @@ lead: Intro text on what is included in this section and how to use it. No more
 
 <div class="preview">
   
-  <h2 class="usa-heading">Sidenav</h2>
-
```

Comment:
Chatted w @colinpmacarthur and we decided on the following language to make the headings a little tighter:

1. Single level 
2. Two level
3. Three level